docs/sql-reference/sql-statements/data-definition/CREATE_TABLE.md

```diff
@@ -633,6 +633,14 @@ PROPERTIES (
   - When this property is set to `true`, the load task returns success as soon as the data is written into the local disk cache, and the data is written into the object storage asynchronously. This allows better loading performance, but it also risks data reliability under potential system failures.
   - When this property is set to `false`, the load task returns success only after the data is written into both object storage and the local disk cache. This guarantees higher availability but leads to lower loading performance.
 
+#### Set fast schema evolution
+
+`fast_schema_evolution`: Whether to enable the fast schema evolution for the table. Valid values are `TRUE` (default) or `FALSE`. Enabling this feature can increase the speed of schema changes and reduce resource usage when columns are added or dropped. Currently, this property can only be enabled at table creation, and it cannot be modified using [ALTER TABLE](../../sql-statements/data-definition/ALTER_TABLE.md) after table creation. This parameter is supported since v3.2.0.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/94ee2ba4-2e11-4329-9336-ac0de87882bc)
or -> and